### PR TITLE
Muhammad Commit second

### PR DESCRIPTION
**Pull request by abbi-code**
All good

### CODE REVIEW

**Comment by ozgurSeker80** [comment]
great

**Comment by alcevik** [comment]
great

File: src/test/java/Test/TestCase1.java [hunk 1]
```diff
@@ -5,5 +5,9 @@
     public static void main(String[] args) {
         // Muhammad branch
 
+
+        // Second commit
```
Comment (abbi-code) [inline src/test/java/Test/TestCase1.java:9]: Remove this

**Review by abbi-code — COMMENTED** [review]
all good

**Review by jhussai73 — APPROVED** [review]
looks good

**Pull request closed without merging**